bool initialized
int32 dmp_id
int32 version
bool is_learned
bool is_setup
bool is_start_set
float64 canonical_x
float64 canonical_time
float64 duration
float64 initial_duration
float64 delta_t
float64 initial_delta_t
string library_directory_name
float64 teaching_duration
float64 execution_duration
float64 can_sys_cutoff
float64 alpha_x
float64 k_gain
float64 d_gain
float64 alpha_z
float64 beta_z
float64 alpha_s
float64 num_samples
dmp_motion_generation/TransformationSystem[] transformation_systems